frontend/src/metabase/visualizations/shared/utils/data.ts

```diff
@@ -149,7 +149,6 @@ export const trimData = (
     {
       dimensionValue: groupedDatumDimensionValue,
       metrics: {},
-      breakout: {},
```

Comment:
It seems reasonable not to create it initially since the chart might not have a break out, so technically speaking having an empty breakout object is not correct logically

Reply:
@alxnddr ignore what I said earlier, you're right. I thought line 139 will always be executed even when `breakout` **doesn't** exist. But that's not the case since it's been guarded by `Object.keys(currentValue.breakout ?? {}).map`.

I'm sorry for confusing you.